### ISSUE TITLE
When I make the SSD, there's some troubles

### ISSUE THREAD

**Issue by ma3252788** [weiliu89/caffe#664]
### Issue summary

src/caffe/layers/video_data_layer.cpp: 在成员函数‘virtual void caffe::VideoDataLayer<Dtype>::DataLayerSetUp(const std::vector<caffe::Blob<Dtype>*>&, const std::vector<caffe::Blob<Dtype>*>&)’中:
src/caffe/layers/video_data_layer.cpp:55:30: 错误： ‘CV_CAP_PROP_FRAME_COUNT’在此作用域中尚未声明
src/caffe/layers/video_data_layer.cpp:60:14: 错误： ‘CV_CAP_PROP_POS_FRAMES’在此作用域中尚未声明
make: *** [.build_release/src/caffe/layers/video_data_layer.o] 错误 1

### Your system configuration
Operating system: Ubuntu14.04
Compiler:
CUDA version (if applicable):8.0
CUDNN version (if applicable):5
BLAS:atlas
Python or MATLAB version (for pycaffe and matcaffe respectively):python 2.7

Thank you for help!

**Comment by ma3252788** [comment]
Anyone can help me?

**Comment by wematan** [comment]
Hi,
After translating the Chinese to English, it seems like it cant find the declarations for OpenCV's CV_CAP_PROP properties. 
Have you installed opencv?
What version?

**Comment by ma3252788** [comment]
Yes ,I have already installed opencv, and the version is  opencv beta 3.0
After testing opencv in darknet can be used.
I have installed caffe before , will it be for this reason?

**Comment by wematan** [comment]
The issue might be with opencv version,
Which version of opencv have you installed?(3 or 2?)
Check which version you have installed and edit the Makefile.config if your using opencv 3.
